src/main/scala/com/wanari/tutelar/providers/oauth2/FacebookService.scala

```diff
@@ -22,9 +22,11 @@ class FacebookService[F[_]: Monad](
   import cats.syntax.functor._
 
   val TYPE            = "facebook"
-  val redirectUriBase = Uri("https://www.facebook.com/v3.2/dialog/oauth")
-  val tokenEndpoint   = Uri("https://graph.facebook.com/v3.2/oauth/access_token")
-  val userEndpoint    = Uri("https://graph.facebook.com/me")
+  val redirectUriBase = Uri("https://www.facebook.com/v4.0/dialog/oauth")
+  val tokenEndpoint   = Uri("https://graph.facebook.com/v4.0/oauth/access_token")
+  val userEndpoint = Uri(
+    "https://graph.facebook.com/v4.0/me?fields=id,first_name,last_name,middle_name,name,name_format,picture,short_name"
```

Comment:
I think we should add only `s"me?fields=$scopes"` or sth like that

Reply:
but if this will not generate error with no permissions this can be a good candidate too

Reply:
current one doesn't generate errors, I tested it with their API explorer. I could move fields into a separate variable if that what you mean.

Reply:
I think this is ok of this really works, as we expect!